src/components/availability-label/helper.ts

```diff
@@ -37,13 +40,23 @@ export const useAvailabilityData = ({
     }
   }, [isOnline]);
 
+  useEffect(() => {
+    // Articles are always available.
```

Comment:
I think we should also have a check if `available` has been set or not.
So in the beginning of `useEffect` we would have a check:
```typescript
if (isAvailable !== null) {
  return;
}
```

Reply:
And then we need `isAvailable` as a dependency for the hook of course...